extensions/replication/failedCRR/FailedCRRProducer.js

```diff
@@ -45,15 +47,18 @@ class FailedCRRProducer {
     /**
      * Publish the given message to the retry Kafka topic.
      * @param {String} message - The message to publish
-     * @param {Function} cb - The callback function
+     * @param {Function} [deliveryReportCb] - called when Kafka
+     * returns a delivery report
      * @return {undefined}
      */
-    publishFailedCRREntry(message, cb) {
+    publishFailedCRREntry(message, deliveryReportCb) {
```

Comment:
I dont see the callback being used anywhere, do we need it?

Reply:
We don't use it today, although I believe it could be used in a future improvement for error management, I though it would be good to keep it as part of the API to make it readily available then.